pkg/validate/validate.go

```diff
@@ -21,6 +23,28 @@ var (
 	}()
 )
 
+// AdditionalTrustBundle checks if the given string contains valid certificate(s) and returns an error if not.
+func AdditionalTrustBundle(v string) error {
+	rest := []byte(v)
+	for {
+		var block *pem.Block
+		block, rest = pem.Decode(rest)
+		if block == nil {
+			return fmt.Errorf("unable to parse certificate, please check the additionalTrustBundle section of install-config.yaml")
```

Comment:
this error is most appropriate in pkg/types/validate where we are validating the field.. 
`invalid block` should be enough here.